bigml/tests/create_ensemble_steps.py

```diff
@@ -74,15 +76,17 @@ def the_ensemble_is_finished_in_less_than(step, secs):
 #@step(r'I create a local Ensemble$')
 def create_local_ensemble(step):
     world.local_ensemble = Ensemble(world.ensemble_id, world.api)
-    world.local_model = Model(world.local_ensemble.model_ids[0])
+    world.local_model = Model(world.local_ensemble.model_ids[0], world.api)
 
 #@step(r'I create a local Ensemble with the last (\d+) models$')
 def create_local_ensemble_with_list(step, number_of_models):
-    world.local_ensemble = Ensemble(world.models[-int(number_of_models):], world.api)
+    world.local_ensemble = Ensemble(world.models[-int(number_of_models):],
+                                    world.api)
 
 #@step(r'I create a local Ensemble with the last (\d+) local models$')
 def create_local_ensemble_with_list_of_local_models(step, number_of_models):
-    local_models = [Model(model) for model in world.models[-int(number_of_models):]]
+    local_models = [Model(model) for model in
```

Comment:
github is displaying this line as if it had lots of whitespace at the end... probably it's just github...